aidies-project/client/src/config.js

```diff
@@ -1,3 +1,3 @@
-export const baseUrl = process.env.REACT_APP_BASE_URL || 'http://localhost:5000';
-export const imageUrl = `${baseUrl}/`;
-export const apiUrl = `${baseUrl}/api`;
+// export const baseUrl = process.env.REACT_APP_BASE_URL || 'http://localhost:5000';
```

Comment:
You can remove this `config.js` file, since your app is using a `proxy` in the `package.json`!